mtest/libmscore/compat114/chord_symbol-ref.mscx

```diff
@@ -5,6 +5,9 @@
     <currentLayer>0</currentLayer>
     <Division>480</Division>
     <Style>
+      <pageWidth>8.5</pageWidth>
```

Comment:
This is caused by a bug fix in my code.  The source file is in Letter size, not A4.  These values _should_ be in the file.